pkg/kubelet/dockertools/fake_docker_client.go

```diff
@@ -175,27 +177,40 @@ func (f *FakeDockerClient) InspectImage(name string) (*docker.Image, error) {
 	return f.Image, err
 }
 
+// Sleeps random amount of time with the normal distribution with given mean and stddev
+// (in milliseconds), we never sleep less than cutOffMillis
+func normalSleep(mean, stdDev, cutOffMillis int) {
+	cutoff := (time.Duration)(cutOffMillis) * time.Millisecond
+	delay := (time.Duration)(rand.NormFloat64()*float64(stdDev)+float64(mean)) * time.Millisecond
+	if delay < cutoff {
+		delay = cutoff
+	}
+	time.Sleep(delay)
+}
+
 // CreateContainer is a test-spy implementation of DockerInterface.CreateContainer.
 // It adds an entry "create" to the internal method call record.
 func (f *FakeDockerClient) CreateContainer(c docker.CreateContainerOptions) (*docker.Container, error) {
 	f.Lock()
 	defer f.Unlock()
 	f.called = append(f.called, "create")
-	err := f.popError("create")
-	if err == nil {
-		f.Created = append(f.Created, c.Name)
-		// This is not a very good fake. We'll just add this container's name to the list.
-		// Docker likes to add a '/', so copy that behavior.
-		name := "/" + c.Name
-		f.ContainerList = append(f.ContainerList, docker.APIContainers{ID: name, Names: []string{name}, Image: c.Config.Image})
-		container := docker.Container{ID: name, Name: name, Config: c.Config}
-		if f.ContainerMap != nil {
-			containerCopy := container
-			f.ContainerMap[name] = &containerCopy
-		}
-		return &container, nil
+	if err := f.popError("create"); err != nil {
+		return nil, err
+	}
+	f.Created = append(f.Created, c.Name)
+	// This is not a very good fake. We'll just add this container's name to the list.
+	// Docker likes to add a '/', so copy that behavior.
+	name := "/" + c.Name
+	f.ContainerList = append(f.ContainerList, docker.APIContainers{ID: name, Names: []string{name}, Image: c.Config.Image})
+	container := docker.Container{ID: name, Name: name, Config: c.Config}
+	if f.ContainerMap != nil {
+		containerCopy := container
+		f.ContainerMap[name] = &containerCopy
+	}
+	if f.EnableSleep {
```

Comment:
nit: making `normalSleep` a method and checks the `f.EnableSleep` there; this avoids checking before use every time.